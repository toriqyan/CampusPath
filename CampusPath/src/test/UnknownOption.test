# Test the output when the user inputs unknown command
ab

a

# Quit the test script
q