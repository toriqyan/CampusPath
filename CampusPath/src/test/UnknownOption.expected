Menu:
	r to find a route
	b to see a list of all buildings
	q to quit

Enter an option ('m' to see the menu): # Test the output when the user inputs unknown command
Unknown option

Enter an option ('m' to see the menu): 
Unknown option

Enter an option ('m' to see the menu): 
# Quit the test script